std/capnp/rpc.capnp

```diff
@@ -645,7 +645,7 @@ struct Disembargo {
   #
   # Message sent to indicate that an embargo on a recently-resolved promise may now be lifted.
   #
-  # Embargos are used to enforce E-order in the presence of promise resolution.  That is, if an
+  # Embargoes are used to enforce E-order in the presence of promise resolution.  That is, if an
```

Comment:
I think I would prefer *not* to fix spelling errors in this file manually, since it's copied over from the reference implementation, and is going to get blown away whenever we pull in changes from upstream. Instead, if these still exist upstream, you could submit a patch. If not, they'll get fixed when we pull in the versions from the next release.

Reply:
Good point, I hadn't realized these were upstream files.  I'll just revert this change.